Update msconvert

chambm opened the pull request:
Update Docker container version
Add mzMLb output format

mzMLb is HDF5 based like mz5 but supports the ion mobility 3 array format (and any arbitrary extra arrays).

chambm committed:
Update msconvert

Update Docker container version
Add mzMLb output format

bgruening commented:
Add such a line: https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/sample/datatypes_conf.xml.sample#L231

You are allowed to add this the last stable release, aka release_25.0

chambm commented:
> You are allowed to add this the last stable release, aka release_25.0

I don't really understand this sentence. But I'll make a PR to add the datatype. Does that mean this PR can't proceed until that PR is merged?

chambm committed:
Merge branch 'master' into chambm-update-msconvert

hechth committed:
Merge branch 'master' into chambm-update-msconvert

mbodova commented:
Hi, we would like to merge this branch since the msconvert on galaxy currently does not process some newer MS accessions. However the tests fail on launching wine64. Can I ask how galaxy uses it so we can maybe locate the bug?

chambm committed:
Change WINE script from wine64 to wine

I think the Wine version that I started using in the container no longer includes the wine64 script.

mbodova commented:
Thank you, it seems that wine is no longer the problem, only failing tests. Are you working on it or would you like someone to have a look at it?:)

chambm commented:
Looking at those failures they are pretty varied. The corrupt RAW thing mirrors another report I got from another user outside Galaxy so I will be looking into that (will probably require an image version update). I don't think I have time to tackle the other at the moment though so if you've got bandwidth, I appreciate it!

mbodova committed:
Minor changes to fix errorrs

mbodova committed:
Updated test files

mbodova committed:
Updated tests and test data to reduce size

chambm committed:
Update version to fix Thermo RAW Wine issue

mbodova committed:
Updated test files for new image version

hechth committed:
Merge pull request #822 from mbodova/review/800

Fix failing tests

hechth committed:
Merge branch 'master' into chambm-update-msconvert

hechth committed:
replace ln -s with cp

hechth committed:
updated file
